File: java/logica_y_validaciones/validar_correo_correcto.java
package logica_y_validaciones;
public class validar_correo_correcto {
    public static void main(String[] args) {
        String correo = "usuario@gmail.com";
        if (correo.contains("@") && correo.contains(".")) {
            System.out.println("Correo válido.");
        } else {
            System.out.println("Correo inválido.");
        }
    }
}

File: visual_basic/condicionales/validar_correo_correcto/validar_correo_correcto.vb
Imports System
Module validar_correo_correcto
    Sub Main(args As String())
        Dim correo As String
        Console.Write("Ingrese el correo electrónico: ")
        correo = Console.ReadLine()
        If correo.Contains("@") And correo.Contains(".") Then
            Console.WriteLine("Correo válido ✅")
        Else
            Console.WriteLine("Correo inválido ❌")
        End If
    End Sub
End Module